binary_search_tree/binary_search_tree.py

```diff
@@ -53,4 +55,9 @@ def contains(self, target):
         # return False
 
     def get_max(self):
-        pass
+        max = self.value
+        while self.right:
+            if self.right.value > max:
```

Comment:
so right here. this will _always_ be true, as long as there is a right child. once there's no right child, you know it's the max (even if a node to the right of the root has a left child, we know that the left child will not be bigger. it'll be bigger than the root, but not bigger than the right child of the root.)

Reply:
That makes sense. Thanks!